zerver/lib/presence.py

```diff
@@ -181,29 +175,7 @@ def get_presence_dict_by_realm(
         )
     )
 
-    mobile_query = PushDeviceToken.objects.distinct("user_id").values_list(
-        "user_id",
-        flat=True,
-    )
-
-    user_profile_ids = [presence_row["user_profile_id"] for presence_row in presence_rows]
-    if len(user_profile_ids) == 0:
-        # This conditional is necessary because query_for_ids
-        # throws an exception if passed an empty list.
-        #
-        # It's not clear this condition is actually possible,
-        # though, because it shouldn't be possible to end up with
-        # a realm with 0 active users.
-        return {}
-
-    mobile_query_ids = query_for_ids(
-        query=mobile_query,
-        user_ids=user_profile_ids,
-        field="user_id",
-    )
-    mobile_user_ids = set(mobile_query_ids)
```

Comment:
This arg was just adding unnecessary queries and dead code and then not being used for anything. Should have been cleaned up in #16381 already probably